code-scanning/properties/scorecards.properties.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "OSSF Scorecards supply-chain security analysis",
+    "creator": "Open Source Security Foundation (OpenSSF) - https://github.com/ossf",
+    "description": "Scorecards is a static analysis tool to assess the security posture of your project",
+    "iconName": "scorecards",
```

Comment:
We'll need an SVG icon matching the name given here. 🙂

Reply:
I sent images to @josepalafox. I think he's tinkering with them to be sure they render well. Our image is https://github.com/ossf/scorecard/blob/main/artwork/openssf_security.svg, which I can add to this PR in the mean time. What would you suggest?

Reply:
I've rendered the setup tile with the image here, to see how that looks. I think with a tighter bounding box (if that's the lingo) it might look slightly better...?
https://github.com/ossf/scorecard/blob/main/artwork/openssf_security_alt.svg

Reply:
I've added the logo to the icons/ folder. I think we're ready to merge. Resolving this issue for now. Let me know if there's something else needed to merge.
Fyi, I'll submit a new PR to update the hash of our action in ~1 week when the final release is available. Would love to hear your thoughts on https://github.com/actions/starter-workflows/issues/1301 which would help reduce the toil and manual effort for both your team and mine.
cc @josepalafox